kube_hunter/modules/hunting/kubelet.py

```diff
@@ -434,6 +452,493 @@ def is_kubesystem_pod(pod):
                     }
 
 
+""" Active Hunters """
+
+
+@handler.subscribe(AnonymousAuthEnabled)
+class FootholdViaSecureKubeletPort(ActiveHunter):
```

Comment:
I would  like to us to call it: `ProveAnonymousAuth`, we have several other proving hunters, in general the architecture of "proving" is to make an active hunter for an already published vulnerability, setting the evidence on the event. such as you did here.

Also, this hunter does not quite set a Foothold, right?

Reply:
Hi @danielsagi ,

I've changed this and you are right, it doesn't set it.